### PR TITLE
SOF-1926 Update AdLibActions and -Pieces rank

### PR DESCRIPTION
**Pull request by LindvedKrvang**
Ranks on AdLibActions and -Pieces are also updated when the Segment ranks are being updated

### CODE REVIEW

File: packages/job-worker/src/ingest/rundownInput.ts [hunk 1]
```diff
@@ -432,6 +437,36 @@ export async function handleUpdatedSegmentRanks(
 	)
 }
 
+function updateAdLibActionRanks(segmentExternalId: string, newRank: number, cache: CacheForIngest): void {
+	const adLibActionsForSegment: AdLibAction[] = cache.AdLibActions.findFetch((adLibAction: AdLibAction) =>
+		adLibAction.externalId.startsWith(segmentExternalId)
+	)
+
+	for (const adLibAction of adLibActionsForSegment) {
+		const oldRankFraction: number = (adLibAction.display._rank ?? 1) % 1
```
Comment (KvelaGorrrrnio) [inline packages/job-worker/src/ingest/rundownInput.ts:446]: Any reason why `?? 1` instead of `?? 0`? I assume that 0 is the intended default value.

**Review by KvelaGorrrrnio — APPROVED** [review]

File: packages/job-worker/src/ingest/rundownInput.ts [hunk 2]
```diff
@@ -432,6 +437,36 @@ export async function handleUpdatedSegmentRanks(
 	)
 }
 
+function updateAdLibActionRanks(segmentExternalId: string, newRank: number, cache: CacheForIngest): void {
```
Comment (KvelaGorrrrnio) [inline packages/job-worker/src/ingest/rundownInput.ts:440]: It would be nice for clarification that `newRank` is the `newSegmentRank`.

**Review by RasmusAlbrektsen — APPROVED** [review]

File: packages/job-worker/src/ingest/rundownInput.ts (hunk 1 above)
Comment (LindvedKrvang) [reply packages/job-worker/src/ingest/rundownInput.ts:446]: `1` was intentional, but mostly to ensure that he rank did have a decimal value. 
I don't think that's a strong argument so I have updated it to be `0`.

File: packages/job-worker/src/ingest/rundownInput.ts (hunk 2 above)
Comment (LindvedKrvang) [reply packages/job-worker/src/ingest/rundownInput.ts:440]: I'm on the fence, but I have updated it.

**Pull request merged**